src/components/buttons/browse-the-book.cjsx

```diff
@@ -21,25 +21,23 @@ module.exports = React.createClass
     unstyled:  React.PropTypes.bool
     bsStyle:   React.PropTypes.string
 
-  getLinkProps: (link) ->
-    linkProps = {target:'_blank', className:'view-reference-guide', href: link}
+  getLinkProps: ->
+    linkProps =
+      className: 'view-reference-guide'
+
     linkProps.className += " #{@props.className}" if @props.className
-    text = @props.children or 'Browse the Book'
+    linkProps.className += " btn btn-#{@props.bsStyle}" unless @props.unstyled
 
```

Comment:
yeah I suppose it should be.  They're still [recommending it](https://facebook.github.io/react/docs/class-name-manipulation.html), and like I said it's something we're already including indirectly thanks to react-bootstrap.